languages/csharp/config.json

```diff
@@ -63,6 +63,18 @@
           "exceptions"
         ]
       },
+      {
+        "slug": "nullability",
+        "uuid": "370e7e54-6a96-11ea-bc55-0242ac130003",
+        "concepts": ["nullability"],
+        "prerequisites": [
+          "strings",
+          "basics",
```

Comment:
This concept can be removed (sorry for suggesting it). We only include the `basics` concept if there are no other concept dependencies.

```suggestion
```